hummingbot/client/ui/parser.py

```diff
@@ -89,11 +89,15 @@ def load_parser(hummingbot, command_tabs) -> [ThrowingArgumentParser, Any]:
                                 dest="precision", help="Level of precions for values displayed")
     history_parser.set_defaults(func=hummingbot.history)
 
-    gateway_parser = subparsers.add_parser("gateway", help="Gateway API configurations")
-    gateway_parser.add_argument("option", nargs="?", choices=("create", "config", "connect", "generate-certs",
-                                                              "status", "test-connection", "start", "stop"), help="Gateway configuration choices")
-    gateway_parser.add_argument("key", nargs="?", default=None, help="Name of the parameter you want to change")
-    gateway_parser.add_argument("value", nargs="?", default=None, help="New value for the parameter")
+    gateway_parser = subparsers.add_parser("gateway", help="Gateway API commands")
+    gateway_parser.add_argument("--create", nargs="?", metavar='', default=argparse.SUPPRESS, help="Create gateway docker container instance")
```

Comment:
Don't change the command format. We should keep it as "gateway create".

As I said in the task ticket, this means we need to use sub-parsers. Do try to get around it by using `add_argument()` - that's not the right solution.